docs/installation/indexing-data/indexing-from-s3.rst

```diff
@@ -41,7 +41,7 @@ take some time, but will add a continental product to your local Datacube.
 
 .. code-block:: bash
 
-    datacube product add https://data.dea.ga.gov.au/geomedian-australia/v2.1.0/product-definition.yaml
-    s3-to-dc --no-sign-request 's3://dea-public-data/geomedian-australia/v2.1.0/L8/**/*.yaml' ls8_nbart_geomedian_annual
+    datacube product add https://explorer.nci.dea.ga.gov.au/products/ga_ls8c_nbart_gm_cyear_3.odc-product.yaml
```

Comment:
Why are we referencing NCI explorer here?  I'm sure the AWS product definition is the same, but we should probably use the AWS explorer anyway.

Reply:
I referenced nci explorer because I noticed that the product yamls used in tests reference nci, so I figured that was the way to go... changed.